client/src/i18n/en/tree.json

```diff
@@ -65,36 +64,36 @@
     "FORMS_MANAGEMENT": "Forms Management",
     "FUNCTION_BONUS_MANAGEMENT":"Function bonus management",
     "GENERAL_LEDGER" : "General Ledger",
-    "GRADE_EMPLOYERS" : "Grade Employees",
     "GRADES" : "Grade Management",
+    "GRADE_EMPLOYERS" : "Grade Employees",
     "HOLIDAYS_MANAGEMENT" : "Holidays Management",
     "HOSPITAL" : "Hospital",
     "HUMANS_RESSOURCES" : "Human Resources",
     "IMPORT_STOCK_FROM_FILE":"Stock Import",
     "INDEXES" : "Indexes",
     "INDICATORS" : "Indicators",
     "INDICATORS_REPORT" : "Indicators report",
+    "INVENTORY" : "Inventory",
     "INVENTORY_ADJUSTMENT":"Inventory Adjustment",
     "INVENTORY_CONFIGURATION" : "Inventory Configuration",
     "INVENTORY_GROUP" : "Group",
-    "INVENTORY" : "Inventory",
     "INVENTORY_LIST" : "Inventory List",
     "INVENTORY_MANAGEMENT" : "Inventory Management",
     "INVENTORY_REGISTRY" : "Inventory Registry",
     "INVENTORY_TYPE" : "Type",
     "INVENTORY_UNIT" : "Dosage Form",
     "INVENTORY_VIEW" : "Stock",
-    "INVOICE_REGISTRY" : "Invoice Registry",
     "INVOICES" : "Invoices",
+    "INVOICE_REGISTRY" : "Invoice Registry",
     "INVOICING_FEES" : "Invoicing Fees",
     "IPR_MANAGEMENT" : "IPR Management",
     "IPR_TAX_CONFIGURATION" : "IPR Tax Configuration",
     "JOURNAL_VOUCHER" : "Journal Voucher",
     "LOCATION" : "Location Management",
     "MONTHLY_BALANCE": "Monthly Balance",
     "MONTHLY_CONSUMPTION" : "[Stock] Monthly consumption",
-    "MULTI_PAYROLL_INDICE" : "Multiple Payroll(indice)",
     "MULTI_PAYROLL" : "Multiple Payroll",
+    "MULTI_PAYROLL_INDICE" : "Multiple Payroll(indice)",
```

Comment:
Let's call this "Multiple Payroll (Index-based)".  It's somewhat a long story at this point.

Reply:
Ok, I will fix this.  Don't release the PR until I update it.

Reply:
Done.  Ready to release.